gatsby-node.js

```diff
@@ -296,3 +296,26 @@ exports.onCreateWebpackConfig = ({ stage, loaders, actions }) => {
         },
     });
 };
+
+const getLearnPageArticles = async () => {
+    const documents = await stitchClient.callFunction('fetchDevhubMetadata', [
+        metadata,
+        {},
+    ]);
+    return documents;
+};
+
+exports.onCreatePage = async ({ page, actions }) => {
+    if (page.path === '/learn/') {
+        const { createPage, deletePage } = actions;
+        const allArticles = await getLearnPageArticles();
+        deletePage(page);
```

Comment:
Weird paradigm for adding page context, but this is what [Gatsby recommends](https://www.gatsbyjs.org/docs/creating-and-modifying-pages/).